<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorMore</class>
 <widget class="QWidget" name="EditorMore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="namePageButton">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="internetPageButton">
       <property name="text">
        <string>Internet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="personalPageButton">
       <property name="text">
        <string>Personal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="customFieldsPageButton">
       <property name="text">
        <string>Custom Fields</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="pageWidget">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
